### PR TITLE
feat: Enable Codespaces with `kit`

### PR DESCRIPTION
**Pull request by alexec**
To try out:

image

Codespaces will start.
When it is ready, run `kit up` in the terminal.
Wait.
Open "ports" tab.
Click on the globe symbol on the webpack row.

### CODE REVIEW

File: tasks.yaml [hunk 1]
```diff
@@ -0,0 +1,89 @@
+metadata:
+  annotations:
+    help: |
+      Install `kit` by following https://github.com/alexec/kit#install.
+      
+      Run `kit up` to start argo.
```
Comment (NikeNano) [inline tasks.yaml:6]: Should we include something in the docs about how to run this as well? Looks really cool.

Comment (alexec) [reply tasks.yaml:6]: You just run `kit up` :)

Comment (NikeNano) [reply tasks.yaml:8]: But how do new users of Argo find out about this? Do we expect them to do read the `task.yaml` file?

Comment (alexec) [reply tasks.yaml:8]: I think we should trial. It is intended to work with Codespaces.

**Review by NikeNano — APPROVED** [review]
LGTM

**Review by isubasinghe — APPROVED** [review]

**Review by sarabala1979 — APPROVED** [review]

**Pull request merged**